ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -316,7 +316,9 @@ class WalletHomeScreen extends React.PureComponent<Props> {
     );
   }
 
-  private helpMessage = (alignCenter: boolean = false): React.ReactNode => (
+  private paymentsIssueBanner = (
```

Comment:
```suggestion
  private renderHelpMessage = (
```


I would just suggest maintaining a naming more similar to the previous naming. It because: 
- in some portions of code, we used to name as "banner" thise objects having a different theme if compared to the text included in the content of the screen
- to maintain a more similar naming if compared to the 'helpMessage' attribute of the TransactionList
- it could be 'helpMessage' paired with 'render' having that it is a function generating a fragment of code

Reply:
Sounds good! Thanks ;) 
renamed in https://github.com/pagopa/io-app/pull/1858/commits/499f631e84b4e04b6649296c66106d07bfd721f2 https://github.com/pagopa/io-app/pull/1858/commits/a5cb11cfa7390bec89b5af4c1cc455faf809b67c